cpp/include/cudf/io/orc.hpp

```diff
@@ -220,7 +220,9 @@ class orc_reader_options {
    *
    * @param val Vector of fully qualified column names.
    */
-  void set_decimal_cols_as_float(std::vector<std::string> val)
+  [[deprecated(
```

Comment:
Does this add a compiler warning when building libcudf?

Reply:
It should issue warnings only when the function is used.
https://en.cppreference.com/w/cpp/language/attributes/deprecated

Reply:
I see. I read that page earlier but thought it was showing compiler output and not runtime output. Thanks for the correction.